### DISCUSSION TITLE
Lesson 1: Unable to deploy first smart contract

### DISCUSSION THREAD

**Discussion by diegosoaresub** [smartcontractkit/full-blockchain-solidity-course-py#658] [Q&A] [answered]
Hello,

I'm trying to deploy my first smart contract to Rinkeby network, but I constantly facing the issue "gas estimation errored" in Remix. I already tried different browsers and MetaMask wallets, but the error is always the same.

I see the message "creation of SimpleStorage pending..." when clicking on "Deploy" button. After several minutes I get the gas estimation error. 

I guess the issue is associated with my MetaMask account. My balance is always 0 ETH, although I can see all transaction in the activity history.

**Network:** Rinkeby
**Address:** [0xCE8af794095a1e0245Edb25a1eC088bBC14195Bb](https://rinkeby.etherscan.io/address/0xce8af794095a1e0245edb25a1ec088bbc14195bb)

Similar issue: https://ethereum.stackexchange.com/questions/110880/couldn-t-see-my-test-eth-on-my-metamask-wallet-balance

**Comment by cromewar** [comment, marked as answer]
Hello @diegosoaresub sorry for the delay, I'll sent you some eth to your wallet now. Also remember you must select the gas price on Remix, try to use the same as on the video. Please let me know if you can do it after I send the Eth.

**Reply by cromewar** [reply]
Sent: https://rinkeby.etherscan.io/tx/0x08c03de1f596b890a727d837db73b5763b142c270a5d76029a95d491c7b47167

**Reply by diegosoaresub** [reply]
Hello @cromewar, thanks for your quick reply.

The error is still the same and my wallet still shows zero balance. It's worth noting that the MetaMask transaction pop-up does not load when I click on Deploy.

Thanks

![This is an image](https://i.ibb.co/xS2dmGj/Screen-Shot-2021-12-30-at-5-21-59-PM.png)


![This is an image](https://i.ibb.co/ZBS32Gx/Screen-Shot-2021-12-30-at-5-24-19-PM.png)

**Reply by cromewar** [reply]
Did you sent me the correct address?? could you sent it over again please?

**Reply by diegosoaresub** [reply]
The address is correct as you can see your transaction in the activity. However, the balance is not reflecting the assets/transactions.

**Reply by cromewar** [reply]
@diegosoaresub could you solve the problem?

**Reply by diegosoaresub** [reply]
@cromewar I was not able to solve the problem. However, I noticed that I can see the proper balance if I use another computer. I believe it has something to do with my browser because it is managed by my organization. They might be blocking something :(

**Reply by cromewar** [reply]
That's weird, try to delete the cache of your browser and reinstalling the extension

**Reply by diegosoaresub** [reply]
@cromewar sorry for the late reply. I used another computer to test the lesson 1. 

I was able to properly deploy the contract using the same MetaMask account, which lead me to believe that my organization is blocking somehow the interaction between Remix and MetaMask. I tried to deploy the contract in three different browsers and I got the same issue.

Do you have any idea about how I can debug this problem?

**Reply by cromewar** [reply]
Jum this is actually tricky, because if is the organization you are working on, the one which blocks those things it could be very hard to solve (and as you already did, using another computer might be the solution) did you talk with your IT department to be sure the problem is on their side?
